frontstage_api/resources/passwords/change_password.py

```diff
@@ -2,7 +2,7 @@
 
 from flask import Response
 from flask import request
-from flask_restplus import Resource
+from flask_restplus import Resource, fields
```

Comment:
To be super picky the two flask imports could be on one line and imports in alphabetical order